deploy/kubernetes/rbac.yaml

```diff
@@ -59,7 +59,8 @@ roleRef:
   apiGroup: rbac.authorization.k8s.io
 
 ---
-# Resizer must be able to work with end point in current namespace
+# Resizer must be able to work with leases based on the leader-election
+# setting.
 # if (and only if) leadership election is enabled
```

Comment:
The full dot at the end of the line breaks the sentence. There's now a dangling part on the next line.

Reply:
sure.. hopefully its correct now :)